### PR TITLE
Pythia fragment for DiBoson Backgoround Samples

### PR DESCRIPTION
**Pull request by jdavis36**
pythia fragment for ZZ_TuneCP5-pythia8 for Run3 analysis

### CODE REVIEW

File: genfragments/ThirteenPointSixTeV/ZZ/ZZToAny_TuneCP5_13p6TeV_pythia8_cfi.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.MCTunes2017.PythiaCP5Settings_cfi import *
```
Comment (menglu21) [inline genfragments/ThirteenPointSixTeV/ZZ/ZZToAny_TuneCP5_13p6TeV_pythia8_cfi.py:4]: from Configuration.Generator.MCTunesRun3ECM13p6TeV.PythiaCP5Settings_cfi import *

File: genfragments/ThirteenPointSixTeV/ZZ/ZZToAny_TuneCP5_13p6TeV_pythia8_cfi.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+from Configuration.Generator.PSweightsPythia.PythiaPSweightsSettings_cfi import *
+
+generator = cms.EDFilter("Pythia8GeneratorFilter",
```
Comment (menglu21) [inline genfragments/ThirteenPointSixTeV/ZZ/ZZToAny_TuneCP5_13p6TeV_pythia8_cfi.py:7]: generator = cms.EDFilter("Pythia8ConcurrentGeneratorFilter",

**Review by menglu21 — COMMENTED** [review]
there is already samples for Run3, [BTV-Run3Summer22EEGS-00015](https://cms-pdmv.cern.ch/mcm/requests?prepid=BTV-Run3Summer22EEGS-00015)

**Comment by jdavis36** [comment]
> BTW, do you really want to generate Pythia-only diboson samples? They usually don't provide a good description of the data -- they are LO to begin with plus have other issues like not modeling off shell decays.

I think that it is used at least for HZZ analysis to model some qqZZ background. We also separately generate other Di-Boson samples to use in conjunction with the pythia samples.

**Comment by jdavis36** [comment]
> there is already samples for Run3, [BTV-Run3Summer22EEGS-00015](https://cms-pdmv.cern.ch/mcm/requests?prepid=BTV-Run3Summer22EEGS-00015)

File: genfragments/ThirteenPointSixTeV/ZZ/ZZToAny_TuneCP5_13p6TeV_pythia8_cfi.py (hunk 2 above)
Comment (jdavis36) [reply genfragments/ThirteenPointSixTeV/ZZ/ZZToAny_TuneCP5_13p6TeV_pythia8_cfi.py:7]: Hi, do you know why BTV did this sample? I was requested by Higgs Gen group to produce these samples. It is fine if they did it, I just want to know if the sample I was requested to make is the same as this one.

**Comment by jdavis36** [comment]
Just Putting this here, I may have misinterpreted what sample was requested of me to generate. HZZ needs specifically 4L decays from this process. I will verify that this is not being produced and then I will change the data cards.

**Comment by Saptaparna** [comment]
I am happy to merge this as long as you know that these are LO samples and we prefer to use NLO exclusive decays of diboson samples when possible.

**Comment by Saptaparna** [comment]
or should I close this now?

**Pull request closed without merging**